MobileCenter/MobileCenter/Internals/Channel/MSChannelDelegate.h

```diff
@@ -14,22 +14,23 @@
 
 /**
  * Callback method that will be called before each log will be send to the server.
- * @param Instance of MSChannel.
```

Comment:
Can you remove these changes? Once we have this commit in `develop`, we can merge back to here.